website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
@lasley you made me read this: http://www.gnu.org/licenses/gpl-faq.html#RequiredToClaimCopyright

So [if you reside in a country adhered to the Berne convention](https://www.firstwriter.com/copyright/berne_convention.shtml), [your copyright gets automatically assigned to you until 50 years after you died](https://en.wikipedia.org/wiki/Berne_Convention#Copyright_term).

Besides that, to publish to OCA you sign the CLA, what makes you a "half copyright" holder. Given this is *GPL-licensed and you only have half of the copyright, and the other half is automatically assigned to you by international laws, I have to agree adding the copyright is irrelevant.

But I don't like removing it when other people have preferred to add it. I'll just change (c) and keep these discoveries in mind for future modules.